### PR TITLE
Build page shows basic log output

### PR DESCRIPTION
**Pull request by nakhbari**
Issue: #709 

This is hard to test since i'm getting 404 errors on every request to the backend while the build runner is running. Workaround: I loaded up the UI before the build ran and paused on the socket request until the build was running.

- [x] I have run `rspec` and corrected all errors
- [x] I have run `rubocop` and corrected all errors
- [x] I have run `npm run test` and corrected all errors
- [x] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving

#### screenshot
image

### CODE REVIEW

File: app/features/build/build_websocket_backend.rb [hunk 1]
```diff
@@ -48,9 +38,9 @@ def call(env)
       ws.on(:open) do |event|
         logger.debug([:open, ws.object_id])
 
-        url_details = fetch_build_details(event)
-        build_number = url_details[:build_number]
-        project_id = url_details[:project_id]
+        params = Rack::Request.new(env).params
+        build_number = params["build"].to_i
+        project_id = params["project"]
```
Comment (taquitos) [inline app/features/build/build_websocket_backend.rb]: Let's be consistent with naming things, `project` and `project_id`, e.g.: `params["project"]` -> `params["project_id"]`
same with `build` and `build_number`.

File: app/features/build/build_websocket_backend.rb [hunk 2]
```diff
@@ -75,9 +65,9 @@ def call(env)
       ws.on(:close) do |event|
         logger.debug([:close, ws.object_id, event.code, event.reason])
 
-        url_details = fetch_build_details(event)
-        build_number = url_details[:build_number]
-        project_id = url_details[:project_id]
+        params = Rack::Request.new(env).params
+        build_number = params["build"].to_i
```
Comment (taquitos) [inline app/features/build/build_websocket_backend.rb]: same here

File: app/features/build/build_websocket_backend.rb [hunk 3]
```diff
@@ -75,9 +65,9 @@ def call(env)
       ws.on(:close) do |event|
         logger.debug([:close, ws.object_id, event.code, event.reason])
 
-        url_details = fetch_build_details(event)
-        build_number = url_details[:build_number]
-        project_id = url_details[:project_id]
+        params = Rack::Request.new(env).params
```
Comment (taquitos) [inline app/features/build/build_websocket_backend.rb]: `params` isn't very descriptive, let's do `request_params` since `params` is overloaded in ruby.

**Comment by taquitos** [comment]
Cool, no implementation concerns here, just naming consistency, and readability.

File: app/features/build/build_websocket_backend.rb (hunk 3 above)
Comment (nakhbari) [reply app/features/build/build_websocket_backend.rb]: fixed

File: app/features/build/build_websocket_backend.rb (hunk 1 above)
Comment (nakhbari) [reply app/features/build/build_websocket_backend.rb]: fixed

File: app/features/build/build_websocket_backend.rb (hunk 2 above)
Comment (nakhbari) [reply app/features/build/build_websocket_backend.rb]: fixed

File: web/app/services/build-log-websocket.service.ts [hunk 4]
```diff
@@ -36,7 +36,7 @@ export class BuildLogWebsocketService {
   }
 
   private createSocketUrl(projectId: string, buildNumber: number) {
-    return `${this.API_ROOT}/projects/${projectId}/builds/${
-        buildNumber}/log.ws`;
+    return `${this.API_ROOT}?project_id=${projectId}&build_number=${
```
Comment (taquitos) [inline web/app/services/build-log-websocket.service.ts]: Oh, do we have access control over this? As in, does this api ensure that the caller actually has permission to connect and stream the output of this build?

Comment (nakhbari) [reply web/app/services/build-log-websocket.service.ts]: I don't think we have any of that yet. This is the legacy method of building. Just getting placeholders here in place until the new agent work is complete.

Comment (taquitos) [reply web/app/services/build-log-websocket.service.ts]: 🆒

**Pull request merged**